src/renderer/features/operations/OperationsConfirm/Withdraw/ui/Confirmation.tsx

```diff
@@ -13,18 +13,24 @@ import { AccountsModal } from '@entities/staking';
 import { useToggle } from '@shared/lib/hooks';
 
 type Props = {
+  id?: number;
   secondaryActionButton?: ReactNode;
   hideSignButton?: boolean;
   onGoBack?: () => void;
 };
 
-export const Confirmation = ({ secondaryActionButton, hideSignButton, onGoBack }: Props) => {
+export const Confirmation = ({ id = 0, secondaryActionButton, hideSignButton, onGoBack }: Props) => {
   const { t } = useI18n();
 
-  const confirmStore = useUnit(confirmModel.$confirmStore);
-  const initiatorWallet = useUnit(confirmModel.$initiatorWallet);
-  const signerWallet = useUnit(confirmModel.$signerWallet);
-  const proxiedWallet = useUnit(confirmModel.$proxiedWallet);
+  const stores = useUnit(confirmModel.$confirmStore);
+  const initiatorWallets = useUnit(confirmModel.$initiatorWallets);
+  const signerWallets = useUnit(confirmModel.$signerWallets);
+  const proxiedWallets = useUnit(confirmModel.$proxiedWallets);
+
+  const confirmStore = stores?.[id];
```

Comment:
Why not [`useStoreMap`](https://effector.dev/en/api/effector-react/usestoremap/)? Should be a lot simpler.
```ts
const confirmStore = useStoreMap({
  store: confirmModel.$confirmStore,
  keys: [id], 
  fn: (x, [id]) => x?.[id]
})
```